client-core/src/wallet/syncer.rs

```diff
@@ -237,8 +234,8 @@ impl<'a, S: SecureStorage, C: Client, D: TxDecryptor> WalletSyncerImpl<'a, S, C,
             SyncState::genesis(env.client.genesis()?.validators)
         };
 
-        let wallet_state = service::load_wallet_state(&env.storage, &env.name, &env.passphrase)?
-            .unwrap_or_default();
+        let wallet_state =
+            service::load_wallet_state(&env.storage, &env.name, &env.enckey)?.unwrap_or_default();
```

Comment:
I don't think this is a good pattern to have. Ideally we should use `Service` objects and their functions. It's ok for now.

Reply:
we can discuss this some time.